src/lib/evil/unposix/sys/meson.build

```diff
@@ -1,17 +1,31 @@
 if target_machine.system() == 'windows'
 
-  evil_unposix_sys_src = ['time.c', 'time.h', 'stat.h', 'types.h']
+  evil_unposix_sys_src = [
+    'time.c',
+  ]
+
+  evil_unposix_sys_header_src = [
+    'time.h', 
+    'stat.h', 
+    'types.h'
+  ]
+
   evil_unposix_sys_lib = library('evil_unposix_sys',
-		                 evil_unposix_sys_src,
-				 #version: 'meson.project_version(),
-		                 install: true,
-				 )
+    evil_unposix_sys_src,
+    #version: 'meson.project_version(),
```

Comment:
I think that this will be uncommented later when this `evil/unposix` become a child from `evil` meson.build. 
For now to be compatible I can uncomment and describe a custom `project_version()`, what do you think?

Reply:
Oh... I have no idea. Would be problematic to have a different version from the main project? In a way, I think it should have its version, but it seems that in EFL every `meson.build` use the same version. @JPTIZ, you have worked more with it, what do you think?

Reply:
Even though Unposix should work as another library, it's best that it follows the same version from EFL (so we don't have to manage multiple versions of a lot of things - and it's also a part of EFL, so makes sense), so meson.project_version should be used.

Reply:
For now I copied from efl meson.build and made it a static value because it seems too much to make it dynamic. I think I'll need to invoke an external command line (probably using regex) inside meson.build to get the version written in efl meson.build file.